typescript/cli/src/utils/chains.ts

```diff
@@ -33,9 +32,9 @@ export async function runSingleChainSelectionStep(
 export async function runMultiChainSelectionStep(
   customChains: ChainMap<ChainMetadata>,
   message = 'Select chains',
-  chainsToFilterOut: ChainName[] = [],
+  chainSelection = false,
```

Comment:
chainSelection is not descriptive really. How about `requireMultiple` or something like that? Also, wha

Reply:
1) What

Reply:
sorry mid-sentence i just asked you in person what `chainsToFilterOut` was used for